lib/presentation/features/favorites/views/favourites_tile_view.dart

```diff
@@ -5,118 +5,51 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:openflutterecommerce/config/theme.dart';
-import 'package:openflutterecommerce/data/abstract/model/product.dart';
-import 'package:openflutterecommerce/presentation/widgets/independent/scaffold_collapsing.dart';
-import 'package:openflutterecommerce/presentation/widgets/widgets.dart';
+import 'package:openflutterecommerce/presentation/widgets/data_driven/favorites_product_tile.dart';
 
-import '../../wrapper.dart';
-import '../favorites_bloc.dart';
+import '../favorite_bloc.dart';
 import '../favorites_event.dart';
 import '../favorites_state.dart';
 
 class FavouritesTileView extends StatefulWidget {
-  final Function({@required ViewChangeType changeType, int index}) changeView;
+  final Function changeView;
```

Comment:
Why did you decide to remove the parameters?

Reply:
They are not required here. The view is changed by state